MBBSEmu/HostProcess/HostRoutines/MenuRoutines.cs

```diff
@@ -250,13 +253,29 @@ private void MainMenuDisplay(SessionBase session, Dictionary<string, MbbsModule>
             var ansiMenuFileName = _configuration.ANSIMenu;
             if (!File.Exists(ansiMenuFileName))
             {
-                session.SendToClient(
-                    "\r\n|GREEN||B|Please select one of the following:|RESET|\r\n\r\n".EncodeToANSIArray());
+                session.SendToClient("\r\n|GREEN||B|Please select one of the following:|RESET|\r\n\r\n".EncodeToANSIArray());
 
-                foreach (var m in modules.Values.OrderBy(x => x.MenuOptionKey.PadLeft(4, '0')))
+                if (modules.Values.Count > 19)
+                {
+                    var moduleList = modules.Values.OrderBy(x => x.MenuOptionKey.PadLeft(4, '0')).Select(m => new Tuple<string, string>(m.MenuOptionKey, m.ModuleDescription)).ToList();
+                    var columnSeed = moduleList.Count / 2;
+                    var columnFlag = moduleList.Count % 2;
+                    if (columnFlag == 1)
+                        columnSeed++;
+                    for (var i = 0; i < columnSeed; i++)
+                    {
+                        if (columnFlag == 0 || i < columnSeed - 1 && columnFlag == 1)
+                            session.SendToClient($"   |CYAN||B|{moduleList[i].Item1,-2}|YELLOW| ... {moduleList[i].Item2,-28}   |CYAN||B|{moduleList[i + columnSeed].Item1,-2}|YELLOW| ... {moduleList[i + columnSeed].Item2,-28}\r\n".EncodeToANSIArray());
```

Comment:
We should reference `ModuleStruct` and make `DESCRP_SIZE` a const with the length (25). This would keep the length constant.

Reply:
Fixed